.github/workflows/pr-lint.yml

```diff
@@ -1,11 +1,13 @@
 name: "Lint Pull Request"
 on: # yamllint disable-line rule:truthy
-  pull_request_target:
-    types: [opened, synchronize, reopened, ready_for_review]
+#   pull_request_target:
```

Comment:
see that github shows a warning here? Maybe this is something that would break it if this linting would be already in main branch. we should comply to it I guess

Reply:
@janiskemper I'm removing the comments. As of now, the workflow is triggering on new PR and subsequent commits to a PR.